smoke-test/src/test/java/org/opennms/smoketest/DatabaseReportIT.java

```diff
@@ -65,19 +63,20 @@
     @Parameterized.Parameters
     public static Object[] data() {
         return new Object[][] {
-                {"EarlyMorningReport", "PDF", "Early morning report", 1},
-                {"ResponseTimeSummaryForNode", "PDF", "Response Time Summary for node", 1},
-                {"AvailabilityByNode", "PDF", "Availability by node", 1},
-                {"AvailabilitySummaryPast7Days", "PDF", "Availability Summary -Default configuration for past 7 Days", 1},
-                {"ResponseTimeByNode", "PDF", "Response time by node", 1},
-                {"SerialInterfaceUtilizationSummary", "PDF", "Serial Interface Utilization Summary", 2},
-                {"TotalBytesTransferredByInterface", "PDF", "Total Bytes Transferred by Interface ", 2},
-                {"AverageAndPeakTrafficRatesForNodesByInterface", "PDF", "Average and Peak Traffic rates for Nodes by Interface", 2},
-                {"InterfaceAvailabilityReport", "PDF", "Interface Availability Report", 2},
-                {"SnmpInterfaceAvailabilityReport", "PDF", "Snmp Interface Availability Report", 2},
-                {"MaintenanceContractsExpired", "PDF", "Maintenance contracts expired", 2},
-                {"MaintenanceContractsStrategy", "PDF", "Maintenance contracts strategy", 2},
-                {"EventAnalysisReport", "PDF", "Event Analysis report", 2},
+                // TODO MVR what about the grafana reports?
```

Comment:
Safe to remove this? Given we've opted not to test these.

Reply:
Yeah, if we don't want to test them I can remove this.